docs/how-we-work.md

```diff
@@ -1,97 +1,100 @@
 
-## Design Research
+## Making Code Changes
 
-We recognize that the Court staff and members of the public will be critical to helping us develop solutions that will deliver better service to petitioners, practitioners, and Court staff. We will continuously perform observational research and usability testing, and create lightweight prototypes, to iterate on designs and develop a system that meets end-user needs.
+### Definition of Done
 
-## Meetings
+There are two checklists that define what "done" is for [user stories](https://github.com/ustaxcourt/ef-cms/blob/staging/.github/ISSUE_TEMPLATE/ustc-story-template.md) and [bug fixes](https://github.com/ustaxcourt/ef-cms/blob/staging/.github/ISSUE_TEMPLATE/bug-report.md).
 
-There are two basic meeting rhythms: daily design calls/standups and bi-weekly agile sprint rituals.
+### Accepting Vendor Work
 
-Every day, at 10 AM Eastern, we hold a design call where the UX/UI designers can present new designs and get feedback from stakeholders and approval from the Product Owner. The development team also demos any new work at the design meeting to get immediate, iterative feedback. The daily design meeting is followed by a tight 15-minute standup.
+Acceptance of work happens through the sprint as work is completed. The procedure is as follows:
 
-Every two weeks we hold sprint ceremonies. Before the start of a sprint, we ensure the backlog is groomed (prioritizing the work in the backlog). Each two-week sprint begins with sprint planning (define the work to be done over the next two weeks). Each sprint ends with a sprint review (demonstrate work done, and accept or reject that work) and a sprint retro (review the process of how the team performed in this sprint).
+1.  Development team completes work — see _Definition of Done_ above
+2.  The product team verifies the functionality against acceptance criteria in a deployed instance
+3.  Development team creates pull request to the Court's staging environment
+4.  The code is reviewed by the Court for compliance with the contract's [performance and deliverable standards](https://github.com/ustaxcourt/case-management-rfq/blob/master/02_SOW.md#deliverables-and-performance-standards), and for general technical quality — see [Code Review Process](https://github.com/ustaxcourt/ef-cms/blob/staging/docs/CODE_REVIEW.md)
+5.  The development team responds to the Court’s code review feedback
+6.  Pull request merged to staging branch and deployed to the staging environment by the Court
+7.  User testing is performed
+8.  Court creates pull request to production branch and merges to update the production environment
 
```

Comment:
We might want to outline the two different Git flows: bugs vs. story work.

Reply:
For this, Pull requests of story work is a PR directly to the `staging` branch after testing has been conducted in the `migration` environment.

For Bugfixes, Pull requests are opened to the `test` environment from work based off of `staging`. If approved, a new PR is made to the `staging` branch and readied for production deploy.